libs/blocks/merch-card-collection/merch-card-collection.js

```diff
@@ -338,5 +335,7 @@ export default async function init(el) {
     'merch-card-collection-render',
     'merch-card-collection-render:start',
   );
+
+  merchCardCollection.shadowRoot.getElementById('resultText')?.setAttribute('aria-live', 'polite');
```

Comment:
@vhargrave is right, we should not update `shadowRoot` from outside.
we should add it at libs/features/mas/src/merch-card-collection.js